client/express-checkout/index.js

```diff
@@ -1,17 +1,23 @@
-/* global jQuery, wcpayExpressCheckoutParams, wcpayECEPayForOrderParams */
+/* global jQuery, wcpayExpressCheckoutParams */
+/**
+ * External dependencies
+ */
 import { __ } from '@wordpress/i18n';
-import { debounce } from 'lodash';
+import { addAction, removeAction } from '@wordpress/hooks';
```

Comment:
All the changes here are coming from `client/tokenized-express-checkout/index.js`